packages/connect-explorer-webextension/README.md

```diff
@@ -0,0 +1,13 @@
+# @trezor/connect-explorer-webextension"
```

Comment:
There are still some types on this line here (")

also please check here - just some inspiraration for connect-explorers readmes https://github.com/trezor/trezor-suite/pull/9212